avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/debrujin/Kmer.scala

```diff
@@ -39,6 +49,15 @@ case class Kmer(val kmerSeq: String, val weight: Int = 1, val inFlank: Boolean =
     prefix + "[" + suffix + "]"
   }
 
+  def toDetailedString: String = {
+    kmerSeq + " * " + multiplicity + ", " + refPos.fold("unmapped")("@ " + _) + "\n" +
```

Comment:
Good point; `mkString` would be preferable with the exception of the `refPos.fold`.